app/src/components/IndicatorFiltersSidebar.vue

```diff
@@ -142,6 +174,18 @@ export default {
     onClickOutside() {
       this.showLayerMenu = false;
     },
+    moveToHighlight(location) {
+      const DEFAULT_PROJECTION = 'EPSG:3857';
+      const geoJsonFormat = new GeoJSON({
+        featureProjection: DEFAULT_PROJECTION,
```

Comment:
This will work for GTIF (as it uses 3857) but would need the current view projection instead to be usable for indicators with different than default mapProjection-> https://github.com/eurodatacube/eodash/blob/staging/app/src/components/map/Map.vue#L423-L430

Reply:
good point, thanks, changed it to use the current map projection